### DISCUSSION TITLE
random movie suggestion

### DISCUSSION THREAD

**Discussion by bawsuser** [jellyfin/jellyfin#7619] [General]
a feature for random movie suggestions would be pretty cool, so that you get 4-5 movies suggested when you have no idea what to watch :)

**Comment by nmcela** [comment]
I think implementing this feature would be worth the effort. I doubt people keep bad movies in their libraries so the random suggestions are likely to be good without any extra algorithm-guesswork-shenanigans required.

If it's a movie I haven't seen yet, there's a reason it's in my library waiting to be watched. Now it's easy for content to get buried and only to be discovered by luck.

**Comment by bradbeattie** [comment]
https://github.com/jellyfin/jellyfin-web/pull/3631 implements random sorting of a library, but I suspect you're looking for random on the _home page_, yeah?

**Comment by loughnane** [comment]
I too would like to see this.
